sandstone/lib/handlers/broadcast.py

```diff
@@ -4,10 +4,24 @@
 import json
 from pydispatch import dispatcher
 
+
+class BroadcastManager(object):
+    _clients = set()
+
+    def add_client(self, client):
+        self._clients.add(client)
+
+    def broadcast(self, message):
+        for client in clients:
+            client.write_message(message)
+
 class BroadcastHandler(tornado.websocket.WebSocketHandler):
```

Comment:
This handler's `on_close()` should call `remove_client()` on the BroadcastManager instance.